2024-08-23.md

```diff
@@ -0,0 +1,34 @@
+Dear Iroha Community,
+
+Here are some highlights on our progress over the past month:
+
+- **Type-safe queries** ([#4833](https://github.com/hyperledger/iroha/pull/4833))
+  - Queries are now split into _iterable_ (return sequences) and _singular_ (return single values) types, with distinct handling for each.
```

Comment:
Don't mention this. It's an implementation detail and we're planning to remove singular queries anyways

Reply:
Good catch, forgot about it